src/Response.php

```diff
@@ -186,6 +180,19 @@ private function setStatusCode($code)
                 static::MAX_STATUS_CODE_VALUE
             ));
         }
+
+        if (! is_string($reasonPhrase)) {
+            throw new InvalidArgumentException(sprintf(
+                'Unsupported HTTP method; must be a string, received %s',
+                (is_object($reasonPhrase) ? get_class($reasonPhrase) : gettype($reasonPhrase))
```

Comment:
You can drop the surrounding parens; they're unneeded when providing an expression as a function argument.